src/locales/en-GB/pages.yaml

```diff
@@ -46,10 +46,12 @@ conductors:
       We are very happy to say that Ruben has been leading Krashna's choir since November 2011.
   orchestra:
     title: >
-      Our orchestra conductor
+      Our orchestra conductor: Stijn Berkouwer
     text: |
-      Due to the unexpected passing of our longtime orchestra conductor, Daan Admiraal, we currently do not have a permanent orchestra conductor. We will update this page with information on his successor, as soon as possible.
+      Stijn has given concerts with the Nederlands Philharmonisch Orkest, the Gelders Orkest, the Noord Nederlands Orkest, Opera Zuid and Opera Lab Barlin and will soon make his debute at the Orkest van het Oosten. Stijn Berkouwer also works with amateur- and studentorchestras on a regular basis, since 2019 he is the chief-conductor of Symphonyorchestra De Philharmonie.
```

Comment:
debute -> debut
studentorchestras -> student orchestras
Symphonyorchestra -> Symphony orchestra

Reply:
It's actually début, as it turns out.